ResearchKit/Common/ORKNavigationContainerView.m

```diff
@@ -139,12 +153,23 @@ - (void)continueButtonAction:(id)sender {
     });
 }
 
+- (void)backButtonAction:(id)sender {
+    // TODO: disable back button for 0.5 sec?
+    [self backAction:sender];
+}
+
 - (void)continueAction:(id)sender {
     ORKSuppressPerformSelectorWarning(
                                       (void)[_continueButtonItem.target performSelector:_continueButtonItem.action withObject:self];
                                       );
 }
 
+- (void)backAction:(id)sender {
+    ORKSuppressPerformSelectorWarning(
+                                      (void)[_backButtonItem.target performSelector:_backButtonItem.action withObject:self];
```

Comment:
Interesting indent. Is this convention?

Reply:
@beekley We could change it, I just followed the style in the above function (`continueAction`). If you look at macro definition for `ORKSuppressPerformSelectorWarning` the ResearchKit authors are wrapping the passed function call into a block and then sending a few compiler commands. Perhaps that's why they went with indenting this way. Objective-C is fun :)